codes/quantum/qubits/quantum_hamming_css.yml

```diff
@@ -0,0 +1,25 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: quantum_hamming_css
+physical: qubits
+logical: qubits
+
+name: '\([[2^r, 2^r-1-2r, 3]]\) Hamming-based CSS code'
+introduced: '\cite{arXiv:0904.2557}'
+
+description: 'Hamming-based CSS code is a special case of CCS code when \(C_1=C_2=[2^r-1,2^r-1-r,3]\) \href{{https://www2.perimeterinstitute.ca/personal/dgottesman/CO639-2004/Lecture5.pdf}{Gottesman Lecture note}, 2004} .'
+
+protection: 'Protects against any single qubit error.'
+
+relations:
+  parents:
+    - code_id: css
+      detail: ''
+
+
+#  cousins:
```

Comment:
Can remove steane here; will add as a child. Please add cousin entry for 
code_id: hamming 
detail: ''

Reply:
Done.